content/zh-cn/designs/gateway/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "网关"
+weight: 50
+---
+
+<!-- {{< synopsis-gateway >}} -->
+充当本地设备之间，以及设备和云之间的中介的设备。
+<!--more-->
+
+## 挑战
+IoT解决方案中的[端点]({{< ref "/glossary/vocabulary#endpoint" >}})通常不具备直接连接互联网的能力，或者是运行在无法直接访问互联网的网络中。即使存在这些限制，从端点获取数据并与这些端点进行交互需要有相应的连通机制
+
+## 解决方案
+IoT解决方案使用"网关"设计来克服端点通常遇到的限制。通过这样的设计网关可以实现与端点的可靠及安全的通信
+
+如下图所示的"网关"设计可以提供这样的功能。图中服务器(Server)是运行在云中。网关(Gateway)则运行在设备可以访问的网络中
+
+![Gateway Design](gateway.png)
+([PPTx](atlas-gateway.pptx))
```

Comment:
This is no longer how the designs should link to PPTx downloads. Instead you can just link to `https://github.com/aws/iot-atlas/blob/master/content/en-us/designs/iot-atlas-patterns.pptx` using `{{< ref path="/designs/iot-atlas-patterns.pptx" lang="en" >}}`. That single PPTx in the English language directory contains all the diagrams of the site's Designs.

Reply:
The link is changed to  /designs/iot-atlas-patterns.pptx